frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -643,7 +643,7 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
             isChecked={enableMonitoring}
             data-checked-state={enableMonitoring}
           />
-          {props.packageManifest.data[0].metadata.labels['opsrc-provider'] !== 'redhat' && (
+          {!props.packageManifest.data[0].metadata.labels.provider?.includes('Red Hat') && (
```

Comment:
We should be rather doing a regexp, checking for `Red Hat` string, since the label could provider label is currently not uniform. I see that the value could be `Red Hat`, `Red Hat Inc.`, `Red Hat performance`, ....
<img width="318" alt="Screenshot 2024-01-10 at 16 21 41" src="https://github.com/openshift/console/assets/1668218/d031b1d3-d97d-433b-90a1-07bafeb9066f">

Reply:
Based on the examples provided, `includes('Red Hat')` should work just fine since `Red Hat` is common across the values.  See See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes